
Exclude invscout command from lsvpd package

invscout command is shipped in IBMInvscout package, which
is not yet available in Fedora. Hence do not compile
invscout command.

Note: Some day, if IBM wants to ship IBMInvscout package in Fedora,
      then we need to modify lspvd spec file to generate IBMInvscout
      package. (similar to upstream lsvpd spec file).

---
diff --git a/Makefile.am b/Makefile.am
index 9e5d4ba..52abb8f 100755
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,7 +12,7 @@ update_h_files = src/include/device.hpp \
 
 INCLUDES = -I./src/include
 
-sbin_PROGRAMS = vpdupdate lsvpd lscfg lsvio lsmcode invscout
+sbin_PROGRAMS = vpdupdate lsvpd lscfg lsvio lsmcode
 
 vpdupdate_SOURCES = src/internal/sys_interface/devicetreecollector.cpp \
 		src/internal/sys_interface/fswalk.cpp \
@@ -43,9 +43,6 @@ lsmcode_SOURCES = src/output/lsmcode.cpp \
 		  src/internal/sys_interface/rtascollector.cpp \
 		$(update_h_files)
 
-invscout_SOURCES = src/output/invscout.cpp \
-		$(update_h_files)
-
 vpdupdate_LDADD = -lz -lstdc++ -lvpd_cxx
 if SGUTIL1
 vpdupdate_LDADD += -lsgutils
@@ -64,8 +61,6 @@ lsvio_LDADD = -lz -lstdc++ -lvpd_cxx
 lsvio_LDFLAGS = -Wall
 lsmcode_LDADD = -lz -lstdc++ -lvpd_cxx
 lsmcode_LDFLAGS = -Wall
-invscout_LDADD = -lz -lstdc++ -lvpd_cxx
-invscout_LDFLAGS = -Wall
 
 AM_CXXFLAGS = -DDEST_DIR='"${exec_prefix}"' -DVPD_VERSION='"@VERSION@"'
 
@@ -97,7 +92,7 @@ install-exec-hook:
 	install -D --mode=644 cpu_mod_conv.conf \
 		${DESTDIR}/etc/lsvpd/cpu_mod_conv.conf
 
-EXTRA_DIST = docs/lsvpd.8 docs/lscfg.8 docs/lsvio.8 docs/lsmcode.8 docs/invscout.8 \
+EXTRA_DIST = docs/lsvpd.8 docs/lscfg.8 docs/lsvio.8 docs/lsmcode.8 \
 		docs/vpdupdate.8 docs/Makefile docs/install.html \
 		bootstrap.sh scsi_templates.conf cpu_mod_conv.conf \
 		Doxyfile bootstrap.sh
diff --git a/docs/Makefile b/docs/Makefile
index 08309db..280607e 100755
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -3,7 +3,7 @@ MAN_STYLE = $(STYLE_PATH)/manpages/docbook.xsl
 XHTML_STYLE = $(STYLE_PATH)/xhtml/docbook.xsl
 FOP_STYLE = $(STYLE_PATH)/fo/docbook.xsl
 
-MANPAGES = lsvpd.8 lscfg.8 lsmcode.8 lsvio.8 vpdupdate.8 invscout.8
+MANPAGES = lsvpd.8 lscfg.8 lsmcode.8 lsvio.8 vpdupdate.8
 HTMLS = install.html
 
 INSTALL = /usr/bin/install -c
